### PR TITLE
change values to pointers to differ missing value from 0

### PR DESCRIPTION
**Pull request by klatys**
For implementation in node_exporter there's a need to differentiate 0 from nil
blocks https://github.com/prometheus/node_exporter/pull/851

### CODE REVIEW

File: sysfs/net_class_test.go [hunk 1]
```diff
@@ -29,35 +29,56 @@ func TestNewNetClass(t *testing.T) {
 		t.Fatal(err)
 	}
 
+	var (
+		AddrAssignType   int64 = 3
```
Comment (mdlayher) [inline sysfs/net_class_test.go]: Very minor nit: all variables in the scope of a function should start with a lowercase letter.

Comment (klatys) [reply sysfs/net_class_test.go]: fixed. thanks!

**Comment by klatys** [comment]
Thanks for the reviews guys, can we merge this? I've got the node_exporter code ready for push, only blocked by this. Thanks a lot!

**Comment by SuperQ** [comment]
Thanks for the help reviewing.

**Pull request merged**